backend/src/app/preprocessing/pipeline/model/pipeline_cargo.py

```diff
@@ -11,14 +11,18 @@ class PipelineCargo(BaseModel):
     ppj_payload: PreprocessingJobPayloadRead = Field(
         description="Parent PreprocessingJobPayload"
     )
-    next_steps: List["PipelineStep"] = Field(
+    ppj_id: str = Field(description="UUID of the PreprocessingJob")
+
+    next_steps: List[
+        SkipValidation
+    ] = Field(  # FIXME: "Hack" to ignore the problems with "PipelineStep"
```

Comment:
please add as comment: FIXME: "Hack" to ignore the cyclic dependency problem with "PipelineStep"